src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const mediumTerm = 3;
+  const mediumTermDiscount = 20;
+  const calc = days * 40;
+
+  if (days < mediumTerm) {
+    return calc;
+  }
+
+  if (days < longTerm) {
+    return calc - mediumTermDiscount;
+  }
+
+  return calc - longTermDiscount;
```

Comment:
The current logic applies the long-term discount when the rental period is 7 days or more, which is correct. However, the medium-term discount is applied only when the days are between 3 and 6. Ensure that the logic correctly reflects the intended discount structure, where the medium-term discount should apply for rentals of 3 to 6 days, and the long-term discount for 7 or more days.